client/js/store.js

```diff
@@ -17,7 +17,23 @@ function detectDesktopNotificationState() {
 	return "blocked";
 }
 
-const store = new Vuex.Store({
+let store = null;
+
+const setMessageNetworkChannel = (message) => {
```

Comment:
I would rather this not exist in `store.js`, or have it move elsewhere in the file and leave `store` as const